Services/AzureManagementAdapter/AzureManagementAdapter.cs

```diff
@@ -18,6 +18,7 @@ namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.Services.AzureManagement
     public interface IAzureManagementAdapterClient
```

Comment:
I think we should have separate classes and not put all the Azure "stuff" here. For instance, we should have
* one class to generate access tokens
* one class to access metrics
* one to manage VMSS
* and in future one to manage IoTHub scale

Reply:
your call... I think we should do the right thing now, adding the new code in the right place, refactoring as needed, so we don't have to get back to this code twice, test twice, etc.